Source/Tools/Flax.Build/Projects/VisualStudio/CSSDKProjectGenerator.cs

```diff
@@ -213,6 +213,22 @@ public override void GenerateProject(Project project, string solutionPath)
                 else
                     csProjectFileContent.AppendLine(string.Format("    <{0} Include=\"{1}\" />", fileType, projectPath));
             }
+
+            if (project.EmbeddedResources != null)
+            {
+                foreach (var file in project.EmbeddedResources)
+                {
+                    var projectPath = Utilities.MakePathRelativeTo(file, projectDirectory);
+                    csProjectFileContent.AppendLine(string.Format("    <None Remove=\"{0}\" />", projectPath));
```

Comment:
This is probably not needed as the generated project files already has the default inclusions disabled with `<EnableDefaultItems>false</EnableDefaultItems>`